feat(blog): mission post (#595)
pyOpenSci/pyopensci.github.io: _posts/2025-03-21-pyopensci-commitment-inclusion.md

---
layout: single
title: "Reaffirming pyOpenSci's Commitment to Inclusion"

> excerpt: "pyOpenSci is growing a thriving, inclusive open source community where everyone—from beginners to experts—can contribute to better, more open science. Join us in breaking down barriers and building a future where all voices shape scientific discovery."

Author: ```suggestion
excerpt: "pyOpenSci is growing a global, vibrant and inclusive open source community where everyone from all backgrounds and identities can contribute to better, more open science. Join us in breaking down barriers and building a future where all voices shape scientific discovery."
```

author: "Executive Council"
authors: "Leah Wasser, Karen Cranston, Tracy Teal"
h2: "pyOpenSci’s mission has always been founded on inclusion"
permalink: /blog/pyopensci-commitment-inclusion.html
header:
  overlay_image: images/headers/pyopensci-floral.png
categories:
  - blog-post
  - community
  - inclusion
classes: wide
toc: true
comments: true
last_modified: 2025-03-21
---

> During challenging times, it’s critical to pause and reflect on who we are, what we care about, and why our work matters. From its inception, pyOpenSci, a US-based fiscally sponsored project, has been committed to actively building an inclusive, welcoming, open source community of practice that supports better, more open science. Our Code of Conduct reinforces our [values,](https://www.pyopensci.org/handbook/CODE_OF_CONDUCT.html) as does the thought that we put into the events that we run, the accessible resources that we develop, and the work that we do to make our [peer review program inclusive](https://www.pyopensci.org/#broadening-participation-in-scientific-open-source) and [creating Python software easier](https://www.pyopensci.org/python-package-guide/tutorials/intro.html).

Author: ```suggestion
During challenging times, it’s critical to pause and reflect on who we are, what we care about, and why our work matters. Since its inception, pyOpenSci, a global community fiscally sponsored in the U.S., has been committed to actively building an inclusive, welcoming, open source community of practice that supports better, more open science. Our Code of Conduct reinforces our [values,](https://www.pyopensci.org/handbook/CODE_OF_CONDUCT.html) as does the thought that we put into the events that we run, the accessible resources that we develop, and the work that we do to make our [peer review program inclusive](https://www.pyopensci.org/#broadening-participation-in-scientific-open-source) and [creating Python software easier](https://www.pyopensci.org/python-package-guide/tutorials/intro.html).
```

## Carving out space for everyone

Creating an inclusive and welcoming space is core to broadening participation in open source and open science.

* When people feel supported, they contribute
* When knowledge is shared, communities grow and thrive.

Through mentorship, beginner-friendly events, and accessible resources, we actively invite those who might not otherwise see a place for themselves in open source. Our review process includes both developers and those who use the software; here, inclusion ensures that our reviews consider usability and accessibility in addition to the technical nuances of packaging.

<figure>
    <picture>
    <source srcset="/images/events/sprint-people-working.webp" type="image/webp">
    <img src="/images/events/sprint-people-working.png" alt="...">
    </picture>
</figure>

Through mentorship and support, we carve out space for new reviewers, which includes those from historically underrepresented backgrounds in open source to contribute to peer review. Similarly, our beginner-friendly lessons and training events are co-developed with beginner-to-expert contributors to ensure the technical concepts are accessible to more people.

> **Inclusion invites everyone to the table**; it creates space for new contributors and supports existing community members. Inclusion creates opportunities for everyone to work together. Experienced contributors–and everyone in between–play crucial roles in sharing knowledge, shaping best practices, and ensuring our work remains relevant to the broader community.

Reviewer: the "Experienced contributors - and everyone in between - " needs clarity about what is at the other end of 'experienced'. Between Experienced and what? Maybe "New contributors, experienced members, and everyone in between"?

> **Inclusion invites everyone to the table**; it creates space for new contributors and supports existing community members. Inclusion creates opportunities for everyone to work together. Experienced contributors–and everyone in between–play crucial roles in sharing knowledge, shaping best practices, and ensuring our work remains relevant to the broader community.

Reviewer 2: ```suggestion
**Inclusion invites everyone to the table**; it creates space for new contributors and supports existing community members. Inclusion creates opportunities for everyone to work together. Together, we share knowledge, shape best practices, and ensure our work serves the broader community.
```
